### ISSUE TITLE
Upgraded aws-amplify from 2.3.0 to 5.0.5 Storage.put errors out "No Cognito Identity pool..."

### ISSUE THREAD

**Issue by JimmySoftLLC** [aws-amplify/amplify-js#10790]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-js/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### JavaScript Framework

React

### Amplify APIs

Storage

### Amplify Categories

storage

### Environment information

<details>

```
# Put output below this line

  System:
    OS: macOS 11.4
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.80 GB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 9.2.0 - ~/Developer/my-trains-list-front-end/node_modules/.bin/npm
  Browsers:
    Chrome: 108.0.5359.124
    Firefox: 79.0
    Safari: 14.1.1
  npmPackages:
    -: ^0.0.1 => 0.0.1 
    @aws-amplify/pubsub: ^4.5.11 => 4.5.11 (5.0.5)
    @aws-sdk/util-base64: ^3.208.0 => 3.208.0 
    @emotion/react: ^11.10.5 => 11.10.5 
    @emotion/styled: ^11.10.5 => 11.10.5 
    @mui/icons-material: ^5.10.14 => 5.10.16 
    @mui/material: ^5.10.14 => 5.10.17 
    @types/react-dom: ^18.0.9 => 18.0.9 
    @types/uuid: ^8.3.4 => 8.3.4 
    @types/validator: ^13.7.10 => 13.7.10 
    aws-amplify: ^5.0.3 => 5.0.5 
    aws-sdk: ^2.1277.0 => 2.1277.0 
    axios: ^1.1.3 => 1.2.1 (0.26.0)
    browser-image-compression: ^2.0.0 => 2.0.0 
    caniuse-lite: ^1.0.30001436 => 1.0.30001439 
    date-fns: ^2.29.3 => 2.29.3 
    g: ^2.0.1 => 2.0.1 
    i: ^0.3.7 => 0.3.7 
    npm: ^9.1.2 => 9.2.0 
    react: ^18.2.0 => 18.2.0 (18.1.0)
    react-dom: ^18.2.0 => 18.2.0 
    react-image-crop: ^10.0.8 => 10.0.9 
    react-image-file-resizer: ^0.4.8 => 0.4.8 
    react-player: ^2.11.0 => 2.11.0 
    serve: ^14.1.2 => 14.1.2 
    tesseract.js: ^3.0.3 => 3.0.3 
    typescript: ^4.9.4 => 4.9.4 
    uuid: ^9.0.0 => 9.0.0 (3.4.0, 3.3.2, 8.3.2, 8.0.0)
    validator: ^13.7.0 => 13.7.0 
  npmGlobalPackages:
    @angular/cli: 10.0.1
    @vue/cli: 5.0.6
    babel-cli: 6.24.1
    gulp-cli: 2.2.0
    heroku: 7.67.1
    i: 0.3.6
    nodemon: 1.19.2
    npm-check-updates: 11.6.0
    npm: 9.1.2
    typescript: 3.9.5
    yarn: 1.19.1

```

</details>

### Describe the bug

Upgraded the aws-amplify library from 2.3.0 to 5.0.5

Storing an image blob using Storage.put but it errors out with "No Cognito Identity pool provided for unauthenticated access".  This was working previously.

### Expected behavior

The blob should be written to S3.

### Reproduction steps

Install Amplify

Create cognito pools, user pool etc 

Create a S3 bucket to write to

Set IAM roles to write the bucket and attach roles to cognito pool

Attempt to write bucket.

### Code Snippet

```javascript
// Calling the manual configuration in this code
import { Storage } from 'aws-amplify';
import consoleLogTimeElapsed from '../../utilities/consoleLogTimeElapsed';
import awsConfig from '../../amplify-config';

Storage.configure(awsConfig);

const uploadImageStorage = async (blob: { type: any }, myId: string) => {
  try {
    let myTimer = new consoleLogTimeElapsed('Upload time amplify');
    let myResult = await Storage.put(myId, blob, {
      level: 'public',
      contentType: blob.type,
    });
    myTimer.timeElapsed();
    return myResult;
  } catch (error) {
    console.log(error);
    return null;
  }
};

export default uploadImageStorage;
```

### Log output

<details>

```
// Put your logs below this line


```

</details>

### Manual configuration

```js
const awsConfig = {
  Auth: {
    region: 'us-east-1',
    userPoolId: 'us-east-1_xxx', 
    userPoolWebClientId: 'xxx', 
    identityPoolId: 'us-east-1:xxx',
  },
  Storage: {
    AWSS3: {
      bucket: 'xxx',
      region: 'us-east-1',
      identityPoolId: 'us-east-1:xxx',
    },
  },
  API: {
    endpoints: [
      {
        name: 'xxx',
        endpoint: 'https://xxx.amazonaws.com/prod',
        region: 'us-east-1',
      },
    ],
  },
};

export default awsConfig;
```

### Additional configuration

```json
{
    "UserPool": {
        "Id": "us-east-1_xxx",
        "Name": "xxx",
        "Policies": {
            "PasswordPolicy": {
                "MinimumLength": 8,
                "RequireUppercase": true,
                "RequireLowercase": true,
                "RequireNumbers": true,
                "RequireSymbols": true,
                "TemporaryPasswordValidityDays": 7
            }
        },
        "DeletionProtection": "INACTIVE",
        "LambdaConfig": {},
        "LastModifiedDate": "2020-06-21T19:26:31.011000+00:00",
        "CreationDate": "2020-03-08T14:55:40.711000+00:00",
        "SchemaAttributes": [
            {
                "Name": "sub",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": false,
                "Required": true,
                "StringAttributeConstraints": {
                    "MinLength": "1",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "name",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "given_name",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "family_name",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "middle_name",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "nickname",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "preferred_username",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "profile",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "picture",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "website",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "email",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": true,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "email_verified",
                "AttributeDataType": "Boolean",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false
            },
            {
                "Name": "gender",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "birthdate",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "10",
                    "MaxLength": "10"
                }
            },
            {
                "Name": "zoneinfo",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "locale",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "phone_number",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "phone_number_verified",
                "AttributeDataType": "Boolean",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false
            },
            {
                "Name": "address",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "0",
                    "MaxLength": "2048"
                }
            },
            {
                "Name": "updated_at",
                "AttributeDataType": "Number",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "NumberAttributeConstraints": {
                    "MinValue": "0"
                }
            },
            {
                "Name": "custom:id",
                "AttributeDataType": "String",
                "DeveloperOnlyAttribute": false,
                "Mutable": true,
                "Required": false,
                "StringAttributeConstraints": {
                    "MinLength": "1",
                    "MaxLength": "256"
                }
            }
        ],
        "AutoVerifiedAttributes": [
            "email"
        ],
        "UsernameAttributes": [
            "email"
        ],
        "SmsVerificationMessage": "Your verification code is {####}. ",
        "EmailVerificationMessage": "Your verification code is {####}. ",
        "EmailVerificationSubject": "Your verification code",
        "VerificationMessageTemplate": {
            "SmsMessage": "Your verification code is {####}. ",
            "EmailMessage": "Your verification code is {####}. ",
            "EmailSubject": "Your verification code",
            "EmailMessageByLink"",
            "EmailSubjectByLink": "Your verification link for xxx",
            "DefaultEmailOption": "CONFIRM_WITH_LINK"
        },
        "SmsAuthenticationMessage": "Your authentication code is {####}. ",
        "MfaConfiguration": "OFF",
        "EstimatedNumberOfUsers": 7,
        "EmailConfiguration": {
            "SourceArn": "arn:aws:ses:us-east-1:xxxx",
            "EmailSendingAccount": "DEVELOPER",
            "From": "xxx"
        },
        "UserPoolTags": {},
        "Domain": "xxx",
        "AdminCreateUserConfig": {
            "AllowAdminCreateUserOnly": false,
            "UnusedAccountValidityDays": 7,
            "InviteMessageTemplate": {
                "SMSMessage": "Your username is {username} and temporary password is {####}. ",
                "EmailMessage": "Your username is {username} and temporary password is {####}. ",
                "EmailSubject": "Your temporary password"
            }
        },
        "UsernameConfiguration": {
            "CaseSensitive": false
        },
        "Arn": "arn:aws:cognito-idp:xxxx",
        "AccountRecoverySetting": {
            "RecoveryMechanisms": [
                {
                    "Priority": 1,
                    "Name": "verified_email"
                },
                {
                    "Priority": 2,
                    "Name": "verified_phone_number"
                }
            ]
        }
    }
}
```

### Mobile Browser

Chrome

**Comment by nadetastic** [comment]
Hi @JimmySoftLLC 👋,

Thanks for opening this issue - I have a few follow up questions on this:

- Are you getting any other errors with different methods, for example `Storage.get`?
- It looks like the Auth resources were configured manually, can you verify that the attached identity pool has an Unauthenticated role and whether under "Unauthenticated identities", `Enable access to unauthenticated identities` is enabled?

**Comment by JimmySoftLLC** [comment]
The s3 bucket is public for get commands so the url is used directly.  
The identify pool has an Unauthenticated role and Enable access to unauthenticated identities is selected.

Still the same error.

Actually the desire is to use an authenticated user to do the Storage.put.  This routine is called after a user is logged in and the same code worked fine with the older library version.

So I am not sure how to make this work.

**Comment by nadetastic** [comment]
Okay thank you for providing additional info. It's interesting that this worked previously, and I'm not quite sure what may have changed from V2 to V5.

You mentioned trying with an authenticated user had no effect, do you get the same or different error message? The original error happens when trying to set guest credentials and no Identity pool ID is configured on the APP.

https://github.com/aws-amplify/amplify-js/blob/dfbabaf54dda902f1f77c4501e78f49e6a9397af/packages/core/src/Credentials.ts#L276-L283

I suspect that there may be some configuration issue here and recommend verifying how your project is configured compared to the documentation at [1]

Specifically, I'd like to confirm that you are passing `awsConfig` to `Amplify.configure(awsConfig)`

[1] https://docs.amplify.aws/lib/client-configuration/configuring-amplify-categories/q/platform/js/#authentication-amazon-cognito

**Comment by nadetastic** [comment]
@JimmySoftLLC following up on this, any updates/changes?

**Comment by JimmySoftLLC** [comment]
@nadetastic Hi Dan sorry for the delay.  

I am currently passing the awsconfig to the Storage.configure(awsconfig) should I be using Amplify.configure(awsConfig) instead?  I will try that and see if it works.

I do a similar thing for my api calls using API.configure(awsconfig) and they work.  The new library required this in every file that api calls were made.  The older library only needed it in my app.js.

**Comment by JimmySoftLLC** [comment]
@nadetastic 

I changed the code to the following and the identify pool error is gone but now I get an error ""Region is missing".  Below is my new code as well as my config file.

```
import { Storage, Auth, Amplify } from 'aws-amplify';
import consoleLogTimeElapsed from '../../utilities/consoleLogTimeElapsed';
import awsConfig from '../../amplify-config';

//Storage.configure(awsConfig);

Amplify.configure(awsConfig);

const uploadImageStorage = async (blob: { type: any }, myId: string) => {
  try {
    let myCredentials = await Auth.currentCredentials();
    let myTimer = new consoleLogTimeElapsed('Upload time amplify');
    let myResult = await Storage.put(myId, blob, {
      level: 'public',
      contentType: blob.type,
    });
    myTimer.timeElapsed();
    return myResult;
  } catch (error) {
    console.log(error);
    return null;
  }
};

export default uploadImageStorage;
```
The config file is as follows.

```
// This file is used for manual configuration of the Amplify library.
// When Amplify is used in conjunction with the Amplify CLI toolchain or AWS Mobile Hub to manage backend resources,
// an aws-exports.js file is auto-generated and can be used instead of the below to automatically configure the Amplify library.
// In this workshop, we are using the Amplify client libraries without the CLI toolchain so you should edit this file manually.

const awsConfig = {
  Auth: {
    region: 'us-east-1', // example: 'us-east-2'
    userPoolId: 'us-east-1_xxxx', // example: 'us-east-2_teEUQbkUh'
    userPoolWebClientId: '2exxx', // example: '3k09ptd8kn8qk2hpk07qopr86'
    identityPoolId: 'us-east-1:xxxx', // example: 'us-east-2:c85f3c18-05fd-4bb5-8fd1-e77e7627a99e'
  },
  Storage: {
    AWSS3: {
      bucket: 'xxxx', // example: 'my-test-bucket'
      region: 'us-east-1', // example: 'us-east-2'
    },
  },
  API: {
    endpoints: [
      {
        name: 'xxxx_api',
        endpoint: 'https://xxx.execute-api.us-east-1.amazonaws.com/prod', // example: 'https://u8swuvl00f.execute-api.us-east-2.amazonaws.com/prod'
        region: 'us-east-1', // example: 'us-east-2'
      },
    ],
  },
};

export default awsConfig;
```

**Comment by JimmySoftLLC** [comment]
I update my config file to have a global definition for s3 region and that has no effect either.

```
const awsConfig = {
  aws_user_files_s3_bucket_region: 'us-east-1', // (required) - Amazon S3 bucket region
  aws_user_files_s3_bucket: 'iwanttodine', // (required) - Amazon S3 bucket URI
  Auth: {
    region: 'us-east-1', // example: 'us-east-2'
    userPoolId: 'us-east-1_4ajU9e1Yn', // example: 'us-east-2_teEUQbkUh'
    userPoolWebClientId: '2e70bconmpetsff86vakbqikn1', // example: '3k09ptd8kn8qk2hpk07qopr86'
    identityPoolId: 'us-east-1:3b9be410-6228-4d44-bf3f-203381691da6', // example: 'us-east-2:c85f3c18-05fd-4bb5-8fd1-e77e7627a99e'
  },
  Storage: {
    AWSS3: {
      bucket: 'iwanttodine', // example: 'my-test-bucket'
      region: 'us-east-1', // example: 'us-east-2'
    },
  },
  API: {
    endpoints: [
      {
        name: 'i_want_to_dine_restaurant_api',
        endpoint: 'https://pf7biibdkk.execute-api.us-east-1.amazonaws.com/prod', // example: 'https://u8swuvl00f.execute-api.us-east-2.amazonaws.com/prod'
        region: 'us-east-1', // example: 'us-east-2'
      },
    ],
  },
};
```

**Comment by nadetastic** [comment]
Interesting that you are getting this new error, Im working to reproduce

However for your initial error  (`"No Cognito Identity pool..."`), I came across another issue where they were upgrading to latest version of amplify and saw the same error:

`" AWSS3Provider-ensure credentials error No Cognito Identity pool provided for unauthenticated access"`

The fix was to delete the `node-modules` folder and re-install the dependencies.

**Comment by JimmySoftLLC** [comment]
I updated my package.json with the following newer libs from npm and the storage now works.  I do get warnings about depreciated items and things that are renamed so there might be more work for aws to do.

```
    "aws-amplify": "^5.0.9",
    "@aws-sdk/util-base64": "^3.208.0",
    "aws-sdk": "^2.1292.0"```

**Comment by JimmySoftLLC** [comment]
> @JimmySoftLLC Interesting that you are getting this new error, Im working to reproduce
> 
> However for your initial error (`"No Cognito Identity pool..."`), I came across another issue where they were upgrading to latest version of amplify and saw the same error:
> 
> `" AWSS3Provider-ensure credentials error No Cognito Identity pool provided for unauthenticated access"`
> 
> The fix was to delete the `node-modules` folder and re-install the dependencies.

I think the new error is from using Amplify.config instead of Storage.config

**Comment by elorzafe** [comment]
@JimmySoftLLC what is your `package.json` file?

**Comment by JimmySoftLLC** [comment]
`{
  "name": "my-trains-list-front-end",
  "version": "1.0.0",
  "description": "Train inventory app",
  "main": "index.js",
  "scripts": {
    "test": "react-scripts test",
    "build": "react-scripts build",
    "start": "react-scripts start",
    "eject": "react-scripts eject"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/JimmySoftLLC/my-trains-list-front-end.git"
  },
  "author": "Jim Bailey",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/JimmySoftLLC/my-trains-list-front-end/issues"
  },
  "homepage": "https://mytrainslist.com",
  "dependencies": {
    "@aws-sdk/eventstream-codec": "^3.226.0",
    "@aws-sdk/util-base64": "^3.208.0",
    "@emotion/react": "^11.10.5",
    "@emotion/styled": "^11.10.5",
    "@mui/icons-material": "^5.10.16",
    "@mui/material": "^5.10.17",
    "aws-amplify": "^5.0.9",
    "aws-sdk": "^2.1292.0",
    "axios": "^1.2.1",
    "browser-image-compression": "^2.0.0",
    "caniuse-lite": "^1.0.30001436",
    "date-fns": "^2.29.3",
    "npm": "^9.1.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-image-crop": "^10.0.8",
    "react-image-file-resizer": "^0.4.8",
    "react-player": "^2.11.0",
    "serve": "^14.1.2",
    "tesseract.js": "^3.0.3",
    "typescript": "^4.9.4",
    "uuid": "^9.0.0",
    "validator": "^13.7.0"
  },
  "overrides": {
    "uuid@>3.4.0 <7": "9.0.0"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@types/react-dom": "^18.0.9",
    "@types/uuid": "^8.3.4",
    "@types/validator": "^13.7.10"
  }
}
`

**Comment by JimmySoftLLC** [comment]
```
{
  "name": "my-trains-list-front-end",
  "version": "1.0.0",
  "description": "Train inventory app",
  "main": "index.js",
  "scripts": {
    "test": "react-scripts test",
    "build": "react-scripts build",
    "start": "react-scripts start",
    "eject": "react-scripts eject"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/JimmySoftLLC/my-trains-list-front-end.git"
  },
  "author": "Jim Bailey",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/JimmySoftLLC/my-trains-list-front-end/issues"
  },
  "homepage": "https://mytrainslist.com",
  "dependencies": {
    "@aws-sdk/eventstream-codec": "^3.226.0",
    "@aws-sdk/util-base64": "^3.208.0",
    "@emotion/react": "^11.10.5",
    "@emotion/styled": "^11.10.5",
    "@mui/icons-material": "^5.10.16",
    "@mui/material": "^5.10.17",
    "aws-amplify": "^5.0.9",
    "aws-sdk": "^2.1292.0",
    "axios": "^1.2.1",
    "browser-image-compression": "^2.0.0",
    "caniuse-lite": "^1.0.30001436",
    "date-fns": "^2.29.3",
    "npm": "^9.1.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-image-crop": "^10.0.8",
    "react-image-file-resizer": "^0.4.8",
    "react-player": "^2.11.0",
    "serve": "^14.1.2",
    "tesseract.js": "^3.0.3",
    "typescript": "^4.9.4",
    "uuid": "^9.0.0",
    "validator": "^13.7.0"
  },
  "overrides": {
    "uuid@>3.4.0 <7": "9.0.0"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@types/react-dom": "^18.0.9",
    "@types/uuid": "^8.3.4",
    "@types/validator": "^13.7.10"
  }
}

```

**Comment by nadetastic** [comment]
@JimmySoftLLC Im glad you were able to get this resolved.

For the error with `Amplify.configure` I haven't been able to reproduce it - since the original issue and been unblocked I'll go ahead and close this issue, but let me know if you have any questions.
